File: Tareas/20/Código RUBY/tarea20.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

sum = 0
prom = 0
c = 0
m = 0
por = 0.0

print("\nIngrese la cantidad de alumnos:\n --> ")
n = gets().to_i

a = Array.new(n)

for i in(0...n)
	print("Ingrese la calificacion del alumno #{i+1} --> ")
	a[i] = gets().to_i
end

for i in(0..n)
	sum = sum + a[i].to_i
	if( a[i].to_i >= 1300 )then
		c = c + 1
		if( a[i].to_i >=1500 )then
			m = m + 1
		end
	end
end

prom = sum/n
por = (c*100)/n

print("\n")
puts("Promedio general del grupo: #{prom}")
puts("Porcentaje de alumnos aprobados: #{por}%")
puts("Numero de alumnos cuya calificacion es mayor o igual a 1500: #{m}")

File: Tareas/20/Código JAVA/tarea20.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea20{
	public static void main (String[] args){
		String entrada;
		int i, n,sum=0, prom=0, c =0, m=0;
		double por=0.0;

		entrada = JOptionPane.showInputDialog(null,"Ingrese la cantidad de alumnos:");
		n = Integer.parseInt(entrada);
		int[] vector = new int[n];

		for(i=0; i<n; i++){
			entrada = JOptionPane.showInputDialog(null,"Calificacion aprobatoria >= 1300\n\nIngrese la calificacion del alumno: "+(i+1));
			vector[i] = Integer.parseInt(entrada);
		}

		for(i=0; i<n; i++){
			sum = sum + vector[i];
			if(vector[i]>=1300){
				c = c + 1;
				if(vector[i]>=1500){ m = m + 1;}
			}
		}
		prom = sum/n;
		por = (c*100)/n;

		JOptionPane.showMessageDialog(null,"Promedio general del grupo: "+prom+"\nPorcentaje de alumnos aprobados: "+por+"%\nNumero de alumnos cuya calificacion es mayor o igual a 1500: "+m);
	} //Fin de función main
} //Fin de clase tarea20